ApollosProject/apollos-apps
# feat: Pull to refresh on the connect tab

## DESCRIPTION

### What does this PR do, or why is it needed?

Two things 

- Adds pull to refresh to the connect tab. Uses a similar strategy as the featureFeeds to refetch multiple children components. 
- Moves the connect tab to a `ConnectScreenConnected` component. Gasp. I might be jumping the horse a bit on this one, but I think it's a start in the right direction. Removes a ton of code from templates: https://github.com/ApollosProject/apollos-templates/pull/291

### How do I test this PR?

Clone the companion PR (https://github.com/ApollosProject/apollos-templates/pull/291). Visit the connect tab. Pull to refresh. You should see your API console register the pull and show you a few queries running (I have a gif on my work computer)
